modules/rest-api/pages/rest-setting-security.adoc

```diff
@@ -0,0 +1,173 @@
+= Manage Encryption Settings
+:page-topic-type: reference
+
+[abstract]
+Establish and retrieve cluster-wide settings for the use of encryption and cipher-suites.
+
+== HTTP method and URI
+
+----
+GET /settings/security
+
+POST /settings/security
+----
+
+== Description
+
+Allows settings to be established for the use of encryption.
+This includes the encryption _level_, for node-to-node encryption within the cluster;
+UI disablement over insecure connections; TLS minimum version; and a list of server-accepted cipher suites.
+
+[#curl-syntax]
+== Curl Syntax
+
+----
+curl -X GET -d hostname=<host>:<port>
+  -d user=<administrator>
+  -d password=<password>
+  -u <administrator>:<password>
+
+curl -X POST -d hostname=<host>:<port>
+  -d user=<administrator>
+  -d password=<password>
+  -u <administrator>:<password>
+  -d disableUIOverHttp=<true|false>
+  -d disableUIOverHttps=<true|false>
+  -d clusterEncryptionLevel=<all|control>
+  -d tlsMinVersion=<tlsv1|tlsv1.1|tslv1.2>
+  -d honorCipherOrder=<true|false>
+  -d cipherSuites=<list-of-accepted-cipher-suites>
+  http://<host>:<port>/settings/security
+----
+
+The syntax for the `POST` method includes the following:
+
+* `disableUIOverHttp`.
+Whether access to Couchbase Web Console should be disabled over http.
+Default is `false`, meaning that the console can be accessed over http.
+
+* `disableUIOverHttps`.
+Whether access to Couchbase Web Console should be disabled over https.
+Default is `false`, meaning that the console can be accessed over https.
+
+* `clusterEncryptionLevel`.
+Controls the level of encryption imposed on inter-node communications within the cluster.
+Can be either `control` (meaning that only server-management information is passed in encrypted form) or `all` (meaning that all information, including data handled by services, is passed in encrypted form).
+
+* `tlsMinVersion`.
+Specifies the minimum TLS version accepted by the cluster.
+Can be `tlsv1`, `tlsv1.1`, or `tlsv1.1`.
+The default is `tlsv1`.
+This parameter can only be set when encryption between cluster-nodes has been explicitly enabled: see xref:manage:manage-nodes/apply-node-to-node-encryption.adoc[Apply Node-to-Node Encryption], for details.
+
+* `honorCipherOrder`.
+Specifies whether the cluster uses its own cipher-suite preference, rather than the client's.
+The cluster's list of accepted cipher-suites can be defined with the `cipherSuites` flag (see below).
+The default value of `honorCipherOrder` is `true`: a setting of `false` is _not_ recommended, since insecure.
+
+* `cipherSuites`.
+Specifies a list of the cipher suites to be used by the cluster, in order of preference.
```

Comment:
Please also add that empty list [] means that ciphers are not set (default value). It means that default set of ciphers will be used.

Reply:
Done.